conans/client/downloaders/file_downloader.py

```diff
@@ -6,8 +6,7 @@
 import six
 
 from conans.client.rest import response_to_str
-from conans.errors import AuthenticationException, ConanConnectionError, ConanException, \
-    NotFoundException, ForbiddenException, RequestErrorException
+from conans.errors import ConanException, NotFoundException, AuthenticationException, ForbiddenException, ConanConnectionError, RequestErrorException
```

Comment:
Your editorconfig seems not working? This lenght is too much.

Reply:
It didn't apply the proper formatting when moving the class (right-click > refactor > move)... 🤷